src/Bicep.LangServer/Handlers/BicepHoverHandler.cs

```diff
@@ -147,6 +146,9 @@ public class BicepHoverHandler : HoverHandlerBase
                     return AsMarkdown(CodeBlockWithDescription(
                         WithTypeModifiers($"param {parameterAssignment.Name}: {declaredParamMetadata.TypeReference.Type}", declaredParamMetadata.TypeReference.Type), declaredParamMetadata.Description));
 
+                case AssertSymbol assert:
+                    return AsMarkdown(CodeBlockWithDescription($"assert {assert.Name}: {assert.Type}", TryGetDescriptionMarkdown(result, assert)));
```

Comment:
Is `assert.Type` always `bool`?

Reply:
Yes. Would you suggest hard coding that value?

Reply:
No, I'm just wondering if there's something more useful we should be displaying instead. I would leave as is unless a good alternative comes to mind